azure-storage-cpp-lite/src/blob/blob_client_wrapper.cpp

```diff
@@ -149,7 +149,7 @@ namespace microsoft_azure {
             }
             catch(const std::exception &ex)
             {
-                std::cerr << ex.what() << std::endl;
+                syslog(LOG_ERR, "Unknown failure in blob_client_wrapper_init.  ex.what() = %s.", ex.what());
                 errno = unknown_error;
```

Comment:
Why is the error set to unknown here if the exception tells exactly what happens here?